[2023-05-07T22:31:00.347+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Summer_Warped_ETL.GetAudioQuality manual__2023-05-07T21:54:25.314545+00:00 [queued]>
[2023-05-07T22:31:00.356+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Summer_Warped_ETL.GetAudioQuality manual__2023-05-07T21:54:25.314545+00:00 [queued]>
[2023-05-07T22:31:00.356+0000] {taskinstance.py:1331} INFO - Starting attempt 8 of 10
[2023-05-07T22:31:00.371+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): GetAudioQuality> on 2023-05-07 21:54:25.314545+00:00
[2023-05-07T22:31:00.378+0000] {standard_task_runner.py:57} INFO - Started process 1750 to run task
[2023-05-07T22:31:00.393+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Summer_Warped_ETL', 'GetAudioQuality', 'manual__2023-05-07T21:54:25.314545+00:00', '--job-id', '1170', '--raw', '--subdir', 'DAGS_FOLDER/warped-dag.py', '--cfg-path', '/tmp/tmpl7u_fh31']
[2023-05-07T22:31:00.395+0000] {standard_task_runner.py:85} INFO - Job 1170: Subtask GetAudioQuality
[2023-05-07T22:31:00.443+0000] {task_command.py:410} INFO - Running <TaskInstance: Summer_Warped_ETL.GetAudioQuality manual__2023-05-07T21:54:25.314545+00:00 [running]> on host 31774baccecb
[2023-05-07T22:31:00.509+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ananta Moharana' AIRFLOW_CTX_DAG_ID='Summer_Warped_ETL' AIRFLOW_CTX_TASK_ID='GetAudioQuality' AIRFLOW_CTX_EXECUTION_DATE='2023-05-07T21:54:25.314545+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-07T21:54:25.314545+00:00'
[2023-05-07T22:31:10.511+0000] {local_task_job_runner.py:299} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2023-05-07T22:31:10.513+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 1750. PIDs of all processes in the group: [1750]
[2023-05-07T22:31:10.514+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 1750
[2023-05-07T22:31:10.514+0000] {taskinstance.py:1540} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-05-07T22:31:10.526+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/extract.py", line 152, in get_song_audio_quality
    r=requests.get(BASE_URL + 'audio-features/' + id, headers=headers)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/adapters.py", line 498, in send
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/airflow/.local/lib/python3.7/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/lib/python3.7/http/client.py", line 1373, in getresponse
    response.begin()
  File "/usr/local/lib/python3.7/http/client.py", line 319, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.7/http/client.py", line 280, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/local/lib/python3.7/socket.py", line 589, in readinto
    return self._sock.recv_into(b)
  File "/usr/local/lib/python3.7/ssl.py", line 1071, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/local/lib/python3.7/ssl.py", line 929, in read
    return self._sslobj.read(len, buffer)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1542, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2023-05-07T22:31:10.534+0000] {taskinstance.py:1373} INFO - Marking task as UP_FOR_RETRY. dag_id=Summer_Warped_ETL, task_id=GetAudioQuality, execution_date=20230507T215425, start_date=20230507T223100, end_date=20230507T223110
[2023-05-07T22:31:10.543+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 1170 for task GetAudioQuality (Task received SIGTERM signal; 1750)
[2023-05-07T22:31:10.566+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=1750, status='terminated', exitcode=1, started='22:30:59') (1750) terminated with exit code 1
